city_scrapers/spiders/il_elections.py

```diff
@@ -0,0 +1,143 @@
+        """
+        `parse` should always `yield` Meeting items.
+
+        Change the `_parse_title`, `_parse_start`, etc methods to fit your scraping
+        needs.
+        """
+        # parse meeting minutes first to associate them with the agenda
+        self._parse_minutes(response)
+
+        # parse the agenda
+        yield scrapy.Request(
+            "https://www.elections.il.gov/AboutTheBoard/Agenda.aspx",
+            callback=self._parse_agenda,
+            dont_filter=True,
+        )
+
+    def _parse_minutes(self, response):
+        """Parse all past board meeting minutes, store for association to meetings."""
+        minutes = response.css("#ContentPlaceHolder1_gvMeetingMinutes td > a")
+        for minute in minutes:
+            date = minute.css("::text").extract_first().strip()
+            parsed_date = datetime.strptime(date, "%a, %B %d, %Y").date()
+            link = minute.css("::attr(href)").extract_first()
+            qs = parse_qs(urlparse(link).query)
+            self.meeting_minutes[parsed_date] = qs["Doc"][0]
+
+    def _parse_addresses(self, response):
+        addr_divs = response.css(".footer-address > div")
+        for addr_div in addr_divs:
+            # remove "Office" from addr_name, e.g. "Springfield Office"
+            addr_name = addr_div.css("p::text").extract_first().split()[0]
+            address_lines = addr_div.css("div > div::text").extract()
+            # skip last two lines where phone and fax is written
+            address = " ".join([line.strip() for line in address_lines[:-2] if line.strip() != ""])
+            self.addresses[addr_name] = address
+
+    def _parse_agenda(self, response):
+        """Parse board agenda."""
+        self._parse_addresses(response)
+
+        meetings = response.css("#ContentPlaceHolder1_gvAgenda > tr")[1:-1]
+        for item in meetings:
+            start = self._parse_start(item)
+            location = self._parse_location(item)
+            meeting = Meeting(
+                title=self._parse_title(item),
+                description=self._parse_description(item),
+                classification=self._parse_classification(item),
+                start=start,
+                end=self._parse_end(item),
+                all_day=self._parse_all_day(item),
+                time_notes=self._parse_time_notes(item),
+                location=location,
+                links=self._parse_links(item, response, start),
+                source=self._parse_source(response),
+            )
+
+            meeting["status"] = self._get_status(meeting)
+            meeting["id"] = self._get_id(meeting)
+
+            yield meeting
+
+    def _parse_title(self, item):
+        """Parse or generate meeting title."""
+        return "Illinois State Board of Elections"
+
+    def _parse_description(self, item):
+        """Parse or generate meeting description."""
+        desc = item.css("td")[3].css("::text").extract_first().strip()
```

Comment:
If you don't need an intermediate variable you can return the output directly without assigning it to `desc`